lib/diff/lcs/change.rb

```diff
@@ -28,7 +28,7 @@ def initialize(*args)
     unless Diff::LCS::Change.valid_action?(@action)
       raise "Invalid Change Action '#{@action}'"
     end
-    raise "Invalid Position Type" unless @position.kind_of? Fixnum
+    raise "Invalid Position Type" unless @position.kind_of? Integer
```

Comment:
Thanks for the review. It seems to me that it is used when we want to change behavior in 2.4 and 2.3 or lower.
However, [Object#kind_of?](http://ruby-doc.org/core-2.4.0/Object.html#method-i-kind_of-3F) _(NOT [Object#instance_of?](http://ruby-doc.org/core-2.4.0/Object.html#method-i-instance_of-3F
))_ has been used here. Therefore, I think that it is better to use `Integer` as it is. `Integer` is a super class of `Fixnum`, so I think that is simple here.